supervised/algorithms/algorithm.py

```diff
@@ -38,15 +40,15 @@ def reload(self):
             self.load(self.model_file_path)
 
     def fit(
-        self,
-        X,
-        y,
-        sample_weight=None,
-        X_validation=None,
-        y_validation=None,
-        sample_weight_validation=None,
-        log_to_file=None,
-        max_time=None,
+            self,
```

Comment:
why are you adding spaces?

Reply:
This is automatic reformatting by the IDE.